### PR TITLE
Add sample for split limit argument

### PR DESCRIPTION
**Pull request by behnammodi**
- added a sample for limit 
- fix some output

### CODE REVIEW

File: 9-regular-expressions/17-regexp-methods/article.md [hunk 1]
```diff
@@ -95,13 +95,19 @@ Splits the string using the regexp (or a substring) as a delimiter.
...
 ```
 
 But we can split by a regular expression, the same way:
 
 ```js run
-alert('12, 34, 56'.split(/,\s*/)) // array of [12, 34, 56]
+alert('12, 34, 56'.split(/,\s*/)) // array of ['12', '34', '56']
+```
+
+Also we can use `limit` for limit output, like this:
```
Comment (vsemozhetbyt) [inline 9-regular-expressions/17-regexp-methods/article.md:107]: Maybe: 'for limited output' or 'to limit output'.

Comment (behnammodi) [reply 9-regular-expressions/17-regexp-methods/article.md:107]: @vsemozhetbyt Thanks, I'll fix it

**Comment by behnammodi** [comment]
@iliakan Why? it's not polite, close PR without explanation.

**Comment by iliakan** [comment]
First, thanks for the PR.

This section is about regular expressions. So more details about `str.split` seems extraneous.

Usually when I ask people to fix their PRs, they don't do that.
That's why I took the first change (array quotes) and applied it in the commit https://github.com/javascript-tutorial/en.javascript.info/commit/6418344d594b08069968f1715472ccfc1dbb4f86.

Sorry for any offence, and thanks for the contribution!

**Comment by behnammodi** [comment]
Thanks for the response, but I think we can add a back-link for `str.split`

**Comment by iliakan** [comment]
We could, but this method is so obvious...

**Comment by behnammodi** [comment]
@iliakan Ok man 😀 , you're right

**Pull request closed without merging**